test/gerkin/tests/features/db_modify.feature

```diff
@@ -0,0 +1,25 @@
+Feature: API endpoint allowing users to create or modify the Digital Registries database schema.
```

Comment:
Similar scenarios to those from https://github.com/GovStackWorkingGroup/bb-digital-registries/pull/5/files#diff-3e14d9809c93905de3a53cd8350c0c1c5a33855fe3181bc5ea59a2bcd8d2340a would fit here as well.